src/Components/DisplayAllArticles.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import axios from "axios";
+import ArticleTemplate from "./ArticleTemplate";
+
+class DisplayAllArticles extends Component {
+  constructor() {
+    super();
+    this.state = {
+      allArticles: []
+    };
+  }
```

Comment:
Please use the simplified constructor if you only set initial state.

Reply:
I do not know what a simplified constructor is.  Can you link to an example?